tests/mocks_test.py

```diff
@@ -410,3 +411,76 @@ def test_import_incidentfields(mocker):
     else:
         assert hasattr(res, 'error')
         assert res.error == 'Partial Error Description'
+
+
+CONFIGURE_TEST_PARAMS = [
```

Comment:
Add a case for no api_key, nor username.
Check that the error is `You must specify either api_key or username/password either as parameters or use env variables`